pkg/runner/ftp_types.go

```diff
@@ -0,0 +1,136 @@
+package runner
+
+import "strings"
+
+type FtpCommand string
+
+const (
+	List FtpCommand = "list"
+	Mkd  FtpCommand = "mkd"
+	Cwd  FtpCommand = "cwd"
+	Pwd  FtpCommand = "pwd"
+	Dele FtpCommand = "dele"
+	Rmd  FtpCommand = "rmd"
+	Mv   FtpCommand = "mv"
+	Cp   FtpCommand = "cp"
+)
+
+type FtpData struct {
+	Path      string `json:"path,omitempty"`
+	Depth     int    `json:"depth,omitempty"`
+	Recursive bool   `json:"recursive,omitempty"`
+	Src       string `json:"src,omitempty"`
+	Dst       string `json:"dst,omitempty"`
+}
+
+type FtpContent struct {
+	Command FtpCommand `json:"command"`
+	Data    FtpData    `json:"data"`
+}
+
+type FtpResult struct {
+	Command FtpCommand  `json:"command"`
+	Success bool        `json:"success"`
+	Code    int         `json:"code,omitempty"`
+	Data    interface{} `json:"data,omitempty"`
```

Comment:
Consider defining a specific type for the `data` field instead of using an interface. Please review